src/linting/checks.jl

```diff
@@ -477,8 +471,24 @@ function check_farg_unused(x::EXPR)
     end
 end
 
+function unwrap_nospecialize(x)
+    is_nospecialize(x) || return x
+    len = length(x)
+    return if len == 2
+        x[2] # unwrapped form
```

Comment:
What does "unwrapped form" mean? `f(@nospecialize x)` vs `f(@nospecialize(x))`?

Reply:
yes, I meant "wrapped" by parentheses or not:
- "unwrapped" form: `@nospecialized args`
- "wrapped" form: `@nospecialized(a = 1)`